docs/docs/userguide/configuration/CONFLUENCE_CONFIGURATION.md

```diff
@@ -18,6 +18,24 @@ For more information, see [Confluence Version Tags](https://hub.docker.com/r/atl
 confluence_version_tag = "<CONFLUENCE_VERSION_TAG>"
 ```
 
+### Number of Confluence application nodes
+
+The initial Confluence installation require to be started only with a single application node. After all the setup steps
+are finished, it is possible to update the `confluence_replica_count` with a number higher than one and run `install.sh` to update
```

Comment:
We can change the replica count to zero, so `higher than one` should change accordingly.

Reply:
Addressed